packages/params/src/forkName.ts

```diff
@@ -91,5 +91,27 @@ export const forkPostElectra = exclude(forkAll, [
   ForkName.deneb,
 ]);
 export function isForkPostElectra(fork: ForkName): fork is ForkPostElectra {
-  return isForkBlobs(fork) && fork !== ForkName.deneb;
+  return isForkPostDeneb(fork) && fork !== ForkName.deneb;
 }
+
+/**
+ * Aliases exported for compatibility. Types and guards above should be used in
```

Comment:
outdated comment

Reply:
That's a new one.  I added that so its clear why the aliases are there

Reply:
ok. i updated the comment so its more clear